### PR TITLE
Fix capitalize function

### PR DESCRIPTION
**Pull request by roggervalf**
This PR fixes the capitalize function since is not using the real value that is passed. Also I'm adding processed on and finished on values in jobDetails

### CODE REVIEW

File: src/server/views/partials/dashboard/jobDetails.hbs [hunk 1]
```diff
@@ -1,16 +1,18 @@
 {{#unless displayJobInline}}
-  <h3>{{ this.id }}</h3>
+<h3>{{ this.id }}</h3>
```
Comment (roggervalf) [inline src/server/views/partials/dashboard/jobDetails.hbs:2]: these changes are only linting changes

File: src/server/views/helpers/handlebars.js [hunk 2]
```diff
@@ -68,11 +68,11 @@ const helpers = {
     return encodeURIComponent(url);
   },
 
-  capitalize(s) {
+  capitalize(value) {
     if (typeof value !== 'string') {
```
Comment (roggervalf) [inline src/server/views/helpers/handlebars.js:72]: the problem is that here, s is not evaluated, so it returns '' string always

**Comment by roggervalf** [comment]
Please @bradvogel when you get a chance, some 👀 on this fix

File: src/server/views/partials/dashboard/jobDetails.hbs [hunk 3]
```diff
@@ -23,48 +25,60 @@
...
-      {{moment this.options.timestamp "llll"}}
+    {{moment this.options.timestamp "llll"}}
     {{/if}}
     {{#if this.timestamp}}
-      {{moment this.timestamp "llll"}}
+    {{moment this.timestamp "llll"}}
     {{/if}}
   </div>
 
+  {{#if this.processedOn}}
```
Comment (roggervalf) [inline src/server/views/partials/dashboard/jobDetails.hbs:35]: these lines were added

File: README.md [hunk 4]
```diff
@@ -1,5 +1,7 @@
 # Arena
 
+[![NPM](https://img.shields.io/npm/v/bull-arena.svg)](https://www.npmjs.com/package/bull-arena) [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg)](https://github.com/prettier/prettier) [![NPM downloads](https://img.shields.io/npm/dm/bull-arena)](https://www.npmjs.com/package/bull-arena) [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
```
Comment (roggervalf) [inline README.md:3]: Maybe it could be nice to have some badgets

File: src/server/views/helpers/handlebars.js (hunk 2 above)
Comment (bradvogel) [reply src/server/views/helpers/handlebars.js:72]: good catch

**Review by bradvogel — APPROVED** [review]

**Pull request merged**